app/models/transcript.rb

```diff
@@ -114,4 +116,10 @@ def retry!
   def _retry=(_val)
     retry!
   end
+
+  # mime type required for rss - not necessarily the Porter detected mime_type we store
+  # https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/1.0.md#attributes
+  def rss_mime_type
+    MIME_TYPES[format&.to_sym]
```

Comment:
just thinking through validation logic for a second for myself. `format` must be present but `mime_type` could be blank (because that's only set from what Porter returns, if anything). and because `format` must always be present, then `rss_mime_type` should always be present; this would only return nil if `format` was blank. (and i guess, if `format` was not a matching key value for `MIME_TYPES`). ok, so this makes sense to me.